lib/ansible/modules/cloud/amazon/route53_health_check.py

```diff
@@ -185,8 +185,16 @@ def health_check_diff(a, b):
         return {}
     diff = {}
     for key in set(a.keys()) | set(b.keys()):
-        if a.get(key) != b.get(key):
-            diff[key] = b.get(key)
+        a_val = a.get(key)
+        b_val = b.get(key)
+        a_type = type(a_val)
+        b_type = type(b_val)
+        if (a_type is int and b_type is unicode):
```

Comment:
Ah, thanks Ryan, I'm not primarily a Python person so I was wondering about that.

I just updated it to run the values through int() but are you saying don't even compare to the type `unicode` in there? Should it be something like this instead?

`if (a_type is int and b_type is not int):` ?